x-pack/plugins/alerting/common/routes/rule/apis/update/schemas/v1.ts

```diff
@@ -11,62 +11,159 @@ import { notifyWhenSchemaV1, alertDelaySchemaV1 } from '../../../response';
+          {
+            start: schema.string({
+              validate: validateHoursV1,
+              meta: { description: 'The start of the time frame in 24-hour notation (`hh:mm`).' },
+            }),
+            end: schema.string({
+              validate: validateHoursV1,
+              meta: { description: 'The end of the time frame in 24-hour notation (`hh:mm`).' },
+            }),
+          },
+          {
+            meta: {
+              description:
+                'Defines the range of time in a day that the action can run. If the `start` value is `00:00` and the `end` value is `24:00`, actions be generated all day.',
+            },
+          }
+        ),
+        timezone: schema.string({
+          validate: validateTimezoneV1,
+          meta: {
+            description:
+              'The ISO time zone for the `hours` values. Values such as `UTC` and `UTC+1` also work but lack built-in daylight savings time support and are not recommended.',
+          },
         }),
-        end: schema.string({
-          validate: validateHoursV1,
-        }),
-      }),
-      timezone: schema.string({ validate: validateTimezoneV1 }),
-    })
+      },
+      { meta: { description: 'Defines a period that limits whether the action runs.' } }
+    )
   ),
 });
 
-export const actionSchema = schema.object({
-  group: schema.maybe(schema.string()),
-  id: schema.string(),
-  params: schema.recordOf(schema.string(), schema.any(), { defaultValue: {} }),
-  frequency: schema.maybe(actionFrequencySchema),
-  uuid: schema.maybe(schema.string()),
-  alerts_filter: schema.maybe(actionAlertsFilterSchema),
-  use_alert_data_for_template: schema.maybe(schema.boolean()),
-});
+export const actionSchema = schema.object(
+  {
+    group: schema.maybe(
+      schema.string({
+        meta: {
+          description:
+            "The group name, which affects when the action runs (for example, when the threshold is met or when the alert is recovered). Each rule type has a list of valid action group names. If you don't need to group actions, set to `default`.",
+        },
+      })
+    ),
+    id: schema.string({
+      meta: { description: 'The identifier for the connector saved object.' },
+    }),
+    params: schema.recordOf(schema.string(), schema.any(), {
+      defaultValue: {},
+      meta: {
+        description:
+          'The parameters for the action, which are sent to the connector. The `params` are handled as Mustache templates and passed a default set of context.',
+      },
+    }),
+    frequency: schema.maybe(actionFrequencySchema),
+    uuid: schema.maybe(
+      schema.string({
+        meta: { description: 'A universally unique identifier (UUID) for the action.' },
+      })
+    ),
+    alerts_filter: schema.maybe(actionAlertsFilterSchema),
+    use_alert_data_for_template: schema.maybe(
+      schema.boolean({
+        meta: {
+          description: 'Indicates whether to use alert data as template or not.',
```

Comment:
It might be nice to include another sentence about what a template is or how it's used.
```suggestion
          description: 'Indicates whether to use alert data as a template.',
```